src/form/bf.jl

```diff
@@ -46,10 +46,56 @@ end
 
 "nothing to do, this model is symmetric"
 function constraint_mc_transformer_power_yy(pm::LPUBFDiagModel, nw::Int, trans_id::Int, f_bus::Int, t_bus::Int, f_idx, t_idx, f_cnd, t_cnd, pol, tm_set, tm_fixed, tm_scale)
+    tm = [tm_fixed[p] ? tm_set[p] : var(pm, nw, p, :tap, trans_id) for p in conductor_ids(pm)]
+
+
+    nph = length(conductor_ids(pm))
+
+    p_fr = [var(pm, nw, :pt, f_idx)[p] for p in f_cnd]
+    p_to = [var(pm, nw, :pt, t_idx)[p] for p in t_cnd]
+    q_fr = [var(pm, nw, :qt, f_idx)[p] for p in f_cnd]
+    q_to = [var(pm, nw, :qt, t_idx)[p] for p in t_cnd]
+
+    w_fr = var(pm, nw, :w)[f_bus]
+    w_to = var(pm, nw, :w)[t_bus]
+
+    for p in 1:nph
+        JuMP.@constraint(pm.model, 1000.0*w_fr[p] == 1000.0*(pol*tm_scale*tm[p])^2*w_to[p])
+    end
+
+    JuMP.@constraint(pm.model, (p_fr + p_to)*1000.0 .== 0)
+    JuMP.@constraint(pm.model, (q_fr + q_to)*1000.0 .== 0)
```

Comment:
What are the reasons for the factors of 1e3? They seem like they can be eliminated cleanly.